tests/entities.go

```diff
@@ -557,3 +566,179 @@ func (entity *configValidatingService) validate(ctx *TestContext, c *gabs.Contai
 		ctx.pathEquals(configs, config.data, path(configType))
 	}
 }
+
+type ca struct {
+	id                        string
+	name                      string                 `json:"name"`
+	isAutoCaEnrollmentEnabled bool                   `json:"isAutoCaEnrollmentEnabled"`
+	isAuthEnabled             bool                   `json:"isAuthEnabled"`
+	isOttCaEnrollmentEnabled  bool                   `json:"isOttCaEnrollmentEnabled"`
+	certPem                   string                 `json:"certPem"`
+	identityRoles             []string               `json:"identityRoles"`
+	tags                      map[string]interface{} `json:"tags"`
+
+	privateKey crypto.Signer     `json:"-"` //utility property, not used in API calls
+	publicCert *x509.Certificate `json:"-"` //utility property, not used in API calls
+}
+
+func newTestCa(identityRoles ...string) *ca {
+	key, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	if err != nil {
+		panic(err)
```

Comment:
Not sure what to do w/ these types of errors. They *should* never happen.

Reply:
I think for those kinds of errors, panic is fine. I would probably pass the test context in and use require.NoError, but only if it was convenient to do so